src/app/components/task/SingleChoiceTask.js

```diff
@@ -115,7 +117,10 @@ class SingleChoiceTask extends Component {
           onClick={submitCallback}
           disabled={this.state.taskAnswerDisabled}
         >
-          <FormattedMessage id="tasks.answer" defaultMessage="Answer Task" />
+          { fieldset === 'tasks' ?
+            <FormattedMessage id="tasks.answer" defaultMessage="Answer Task" /> :
+            <FormattedGlobalMessage messageKey="save" />
+          }
```

Comment:
Another note for the future... we can refactor and reuse code better for all those task answer components.